gapic/res/text/keyboard-mouse-help.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+<head i18n-values="dir:textdirection;">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta charset="utf-8"/>
+<title>Keyboard/Mouse Help</title>
+<body>
+   <style>
+    :host {
+      flex: 1 1 auto;
+      flex-direction: row;
+      display: flex;
+      width: 700px;
+    }
+    .pair {
+      flex: 1 1 auto;
+      flex-direction: row;
+      display: flex;
+    }
+    .column {
+      padding: 0 40px 40px 0;
+      vertical-align: top;
+    }
+    .command {
+      font-family: monospace;
+      display:inline;
+      margin-right: 5px;
+      text-align: right;
+      min-width: 200px;
+    }
+    .action {
+      font-size: 0.8em;
+      display:inline;
+      text-align: left;
+      min-width: 400px;
+    }
+    h4 {
+      font-weight: bold;
+      font-size: 0.9em;
+    }
+
+    </style>
+
+    <table>
+    <tr>
+     
+    <td class="column">
+      <h2>General</h2>
+      <div class="pair">
+        <div class="command">1-4</div>
```

Comment:
Can you please turn this into a table, so that all the first elements of each pair has the same width, so that the descriptions line up. Right now it's a zig-zagg-y mess.